Calculator.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+
+int main()
+{
+    char s[100];
+    printf("Enter the expression for evaluation: ");
+    scanf("%[^\n]",s);
+
+    int arr_num[100],index_num=0,index_op=0;
+    char arr_op[100];
+    int digit=0;
+    int i,j,k;
+    for(i=0;s[i]!='\0';i++)
+    {
+        char ch=s[i];
+        if(isspace(ch))
```

Comment:
Don't directly use inbuilt functions isSpace and isDigit, create your own custom implementation of these.